src/App.js

```diff
@@ -1,24 +1,70 @@
 import React from 'react';
 import './App.scss';
 
-const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+class App extends React.Component {
+  state = {
+    goodsFromServer: [
+      'Dumplings',
+      'Carrot',
+      'Eggs',
+      'Ice cream',
+      'Apple',
+      'Bread',
+      'Fish',
+      'Honey',
+      'Jam',
+      'Garlic',
+    ],
+    selectedGood: 'Jam',
+  }
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+  render() {
+    const { goodsFromServer, selectedGood } = this.state;
+
+    return (
+      <div className="App">
+        {selectedGood ? (
+          <h1>
+            {selectedGood}
+            {' '}
+            is selected
+            <button
+              type="button"
+              onClick={() => {
+                this.setState({
+                  selectedGood: null,
+                });
+              }}
+            >
+              x
+            </button>
+          </h1>
+        ) : (
+          <h1>No goods selected</h1>
+        )}
+        {goodsFromServer.map(good => (
+          <div className={selectedGood === good ? 'active' : 'good'} key={good}>
+            {good}
+            {selectedGood === good ? (''
+            ) : (
+              <button
```

Comment:
```suggestion
            {selectedGood !== good ?? (
              <button
```
Let's use proper conditional rendering